parsel/__init__.py

```diff
@@ -9,6 +9,13 @@
 
 from parsel.selector import Selector, SelectorList  # NOQA
 from parsel.csstranslator import css2xpath  # NOQA
-from parsel import xpathfuncs # NOQA
+from parsel import xpathfuncs  # NOQA
+
+__all__ = [
+    'Selector',
+    'SelectorList',
+    'css2xpath',
+    'xpathfuncs',
+]
```

Comment:
This is to mention to mypy that the import above are actually meant to be be re-exported. see https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport

Reply:
Oh, I see. What about adding `--implicit-reexport` to the `mypy` call in `tox.ini` instead? I don’t think this is something we want to check for, at least at the moment.

Reply:
This would work for us, but would be problematic for users of the library. Any users running mypy on code that relies on `parsel` would then need to add the flag as well.

Reply:
I must say I don’t like that mypy is forcing us to make this style change just to not break stuff for other people. But given it’s a trivial change, I guess we can budge.

In that case, I would only suggest to place `__all__` before imports, to [comply with PEP8](https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names).

Reply:
Done.